packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -111,53 +97,20 @@ PropsWithChildren<Props>) => {
               {child}
             </ListItem>
           ))}
-          {/* {tags && tags.length > 0 && (
-            <ListItem alignItems="flex-start">
-              <StackOverflow>
-                {tags.map((tag, i) => (
-                  <StackOverflowTag key={i} tag={tag} />
-                ))}
-              </StackOverflow>
-            </ListItem>
-          )} */}
-          {slackChannels && (
-            <ListItem>
-              <ListItemIcon>
-                <GroupIcon />
-              </ListItemIcon>
-              <ListItemText
-                disableTypography
-                primary={<Typography>Support</Typography>}
-                secondary={
-                  <div>
-                    {slackChannels.map((channel, i) => (
-                      <Link key={i}>{channel}</Link>
-                    ))}
-                  </div>
-                }
-              />
-            </ListItem>
-          )}
-          {contactEmails.length > 0 && (
-            <ListItem>
-              <ListItemIcon>
-                <GroupIcon />
-              </ListItemIcon>
-              <ListItemText
-                disableTypography
-                primary={<Typography>Contact</Typography>}
-                secondary={
-                  <div>
-                    {contactEmails.map((em, index) => (
-                      <Typography key={index}>
-                        <Link>{em}</Link>
-                      </Typography>
-                    ))}
-                  </div>
-                }
-              />
-            </ListItem>
-          )}
+          <ListItem>
+            <ListItemIcon>
+              <GroupIcon />
+            </ListItemIcon>
+            <ListItemText
+              disableTypography
+              primary={<Typography>Support</Typography>}
+              secondary={
+                <div>
+                  <Link to={support.url}>{support.title}</Link>
+                </div>
```

Comment:
is this div wrapper necessary?

Reply:
Not really, I've re-worked this and it uses a react fragment now + a list of links).